.github/ISSUE_TEMPLATE/general_issue.md

```diff
@@ -0,0 +1,12 @@
+---
+name: general_issue
+about: Default issue template
+title: Feedback or issue summary
+labels: ''
+assignees: ''
+
+---
+
+**W&B employees**: Please file a DOCS JIRA instead.
```

Comment:
```suggestion
**W&B employees**: Please [file a DOCS JIRA](https://wandb.atlassian.net/secure/CreateIssueDetails!init.jspa?priority=3&pid=10026&issuetype=10047) instead.
```

Reply:
Will this even render in Markdown, though? If this templates presents as raw editable text maybe we should jettison the asterisks, braces, and parentheses?